### PR TITLE
Move resource to SDK.

### PR DESCRIPTION
**Pull request by bogdandrutu**
All updates to the specs related to https://github.com/open-telemetry/oteps/blob/master/text/0007-no-out-of-band-reporting.md. After this PR is merged the otep can probably be marked as "implemented".

### CODE REVIEW

File: specification/library-layout.md [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
 ## API Package
 Here is a proposed generic package structure for OpenTelemetry API package.
 
+A typical top-level directory layout:
+```
+api
+   ├── context
+   │   └── propagation
+   ├── metrics
+   ├── trace
```
Comment (songy23) [inline specification/library-layout.md]: `trace` also has `propagation`.

Comment (bogdandrutu) [reply specification/library-layout.md]: done.

**Comment by Oberon00** [comment]
Note that the latest revision of https://github.com/open-telemetry/oteps/pull/16 will require resources again at the API level.

**Comment by bogdandrutu** [comment]
@Oberon00 thanks for pointing me to that :), just added some comments about this concern there.

File: specification/sdk-resource.md [hunk 2]
```diff
@@ -8,17 +8,12 @@ with closed source environments. API MUST allow for creation of `Resources` and
...
-`Resource` that is set on `Tracer` MUST be ignored. Note, that association of
-`Tracer` with the `Resource` will be done in SDK, not as API call.
 
-**TODO**: explain how resource is associated with metrics.
-
-## Resources SDK
-
-TODO: notes how Resources API is extended when using `SDK`. https://github.com/open-telemetry/opentelemetry-specification/issues/61 
+When used with distributed tracing, a resource can be associated with the
+[Meter](sdk-metrics.md#meter-sdk). When associated with `Meter`, all `Metrics`
```
Comment (SergeyKanzhelev) [inline specification/sdk-resource.md]: how can it be associated with `Meter` if `Meter` is an API concept?

**Comment by Oberon00** [comment]
Probably we have to rethink that association with https://github.com/open-telemetry/oteps/pull/42

File: specification/sdk-resource.md (hunk 2 above)
Comment (bogdandrutu) [reply specification/sdk-resource.md]: Tried to clarify. You do associate with MeterSDK which is the implementation of the Meter so all the metrics created by the MeterSdk via the Meter interface are associated with that resource.

**Comment by bogdandrutu** [comment]
@SergeyKanzhelev Meter is just an interface, you associate with the implementation of the interface so "indirectly" you associate with the Meter.

**Pull request merged**